proto/streamlit/proto/Widget.proto

```diff
@@ -48,5 +48,6 @@ message FloatArray {
 }
 
 message IntArray {
+  // Todo: use sint64
```

Comment:
what about this todo?

Reply:
It's really for the slider. I'll create an issue for that and remove the comment.

Reply:
gotcha!